nixd/lib/Controller/Hover.cpp

```diff
@@ -87,6 +87,10 @@ class NixpkgsHoverProvider {
     return OS.str();
   }
 
+  static std::string mkValueMarkdown(const ValueDescription &ValueDesc) {
+    return ValueDesc.Doc;
```

Comment:
here, the documentation is replied directly without any formatting/tokenization.